rpcs/blockService.go

```diff
@@ -295,18 +301,27 @@ func (bs *BlockService) redirectRequest(round uint64, response http.ResponseWrit
 
 	peers := bs.net.GetPeers(network.PeersPhonebookArchivers)
 	if len(peers) == 0 {
-		return fmt.Errorf("redirecRequest: no arcihver peers found")
+		return fmt.Errorf(errorNoRedirectPeers)
```

Comment:
since these are const objects, why don't you use errors.New() when defining the `const` and avoid declaring it here ?